packages/lodestar/test/e2e/noEth1Sim.test.ts

```diff
@@ -68,21 +72,29 @@ describe("no eth1 sim (multi-node test)", function () {
         }
       }
 
+      // wait for all nodes to be synced first or Validator API would failed
```

Comment:
Isn't it the goal for api to fail until we are synced?

Reply:
Maybe we can just put warning in VC instead of error in case of syncing

Reply:
I removed this, validator is handling pregenesis nicely with logs like `Waiting for genesis time waitTime=26s`